Add `osu!catch World Cup 2024: Round of 16 Recap` news post (#11531)
ppy/osu-wiki: news/2024/2024-06-06-osu-catch-world-cup-2024-round-of-16-recap.md

---
layout: post
title: "osu!catch World Cup 2024: Round of 16 Recap"
date: 2024-06-06 12:00:00 +0000
---

> Welcome to the recap of the Round of 16!

Reviewer: might be a nitpick but the intro feels a bit too, shallow?
```suggestion
It's that time of the week again, grab yourself a fruit of your choice and join us in remembering everything that happened in the osu!catch World Cup 2024 Round of 16!
```

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2161794497&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

Japan and the Russian Federation are closing out the weekend and this recap with one of the closest matches of the entire weekend. Japan, seed 8, are facing off their seed neighbours, seed 9 the Russian Federation. These seed neighbour matches are most likely to bring out very close matches and this one isn't the exception to the rule.

Starting off this match, the Russian Federation won the first map picked by Japan, [HD2](https://osu.ppy.sh/beatmapsets/2187441#fruits/4624959). Japan then proceeded to break the Russian Federation back on [HR2](https://osu.ppy.sh/beatmapsets/2187388#fruits/4624822). Both teams follow this with point wins on their own picks. As their third pick of the match, Japan opted to go for the convert [NM4](https://osu.ppy.sh/beatmapsets/2187368#osu/4624782) but they weren't prepared for the 2.8 million team score the Russian Federation produced.

> It was now vital for Russia to confirm this break point with a map win on their own pick, [NM1](https://osu.ppy.sh/beatmapsets/2187346#fruits/4624746). Three quarters in, [Kimitakari](https://osu.ppy.sh/users/4741164) disconnected from the stream and thus there was a lot of confusion regarding which team actually won the point. After all the dust settled, it was proven Kimitakari's score was high enough to secure Russia the point. They also won [MM1](https://osu.ppy.sh/beatmapsets/2187434#fruits/4624950) to get themselves to match point. Japan, three points down on Russia, had a spirited comeback winning both [HR1](https://osu.ppy.sh/beatmapsets/2187385#fruits/4624818) and [HR3](https://osu.ppy.sh/beatmapsets/2187382#fruits/4624812) before the Russian Federation closed out the match 6-4 on [DT3](https://osu.ppy.sh/beatmapsets/2187391#fruits/4624826). The Russian Federation now guarantee themselves a top 12 position and a place in the winner's bracket quarterfinals.

Reviewer 2: ```suggestion
It was now vital for Russia to confirm this break point with a map win on their own pick, [NM1](https://osu.ppy.sh/beatmapsets/2187346#fruits/4624746). Three quarters in, [Kimitakari](https://osu.ppy.sh/users/4741164) disconnected from the stream, causing lot of confusion regarding which team actually won the point. After all the dust settled, it was proven Kimitakari's score was high enough to secure Russia the point. They also won [MM1](https://osu.ppy.sh/beatmapsets/2187434#fruits/4624950) to get themselves to match point. Japan, three points down on Russia, had a spirited comeback winning both [HR1](https://osu.ppy.sh/beatmapsets/2187385#fruits/4624818) and [HR3](https://osu.ppy.sh/beatmapsets/2187382#fruits/4624812) before the Russian Federation closed out the match 6–4 on [DT3](https://osu.ppy.sh/beatmapsets/2187391#fruits/4624826). The Russian Federation is now guaranteed a top 12 position and a place in the winners' bracket Quarterfinals.
```

---

> Thank you very much for tuning into this week's osu!catch World Cup recap. As always, if you are curious about the full tournament details, the [wiki page](/wiki/Tournaments/CWC/2024) has you covered, and you can *catch* the streams live over on the [osulive Twitch channel](https://www.twitch.tv/osulive)!

Reviewer 2: different outro from last time

```suggestion
The osu!catch World Cup is ramping up both in production quality with fully custom-made maps as well as the length of matches as scorelines become closer. The full tournament details and schedules for anyone following along can be found on the [wiki page](/wiki/Tournaments/CWC/2024), and we'll see you on the [osulive Twitch channel](https://www.twitch.tv/osulive) for the Round of 32!
```
